servers/cromwell/README.md

```diff
@@ -38,45 +38,52 @@ Thin shim around [`cromwell`](https://github.com/broadinstitute/cromwell).
     {
       "field": "labels.cromwell-workflow-id",
       "display": "Workflow ID"
-    },
-    {
-      "field": "labels.flag",
-      "display": "Flag",
-      "editable": true,
-      "bulkEditable": true,
-      "fieldType": "list",
-      "validFieldValues": [
-        "archive",
-        "follow-up"
-      ]
-    },
-    {
-      "field": "labels.label",
-      "display": "Label",
-      "fieldType": "text",
-      "editable": true,
-      "bulkEditable": true
-    },
-    {
-      "field": "labels.comment",
-      "display": "Comment",
-      "fieldType": "text",
-      "editable": true
     }
   ],
   "commonLabels": [
     "cromwell-workflow-id",
-    "workflow-name",
-    "flag"
+    "workflow-name"
   ],
   "queryExtensions": []
 }
 ```
-Both "editable" and "bulkEditable" will be treated as `false` unless explicity set to `true`; if the field is editable, then "fieldType" is required.
```

Comment:
I think we should still include an example of editable/bulkEditable fields somewhere in the readme

Reply:
Oops, something wrong during the rebasing I think, good catch!